.travis.yml

```diff
@@ -3,6 +3,5 @@
 
 language: go
 
-# We can't build with 1.4.x (apparently the Travis default as of 2015-09-30),
-# because it doesn't have http.Request.Cancel.
-go: 1.5.1
+# Updating go version for dependencies on context and net/http/trace
```

Comment:
This comment doesn't make sense to me -- it sounds more like a commit message. Remove it? Or revert it? I'm not sure I understand why you've changed it.

Reply:
Only changed because the travis build was failing due to missing packages "context" and "net/http/trace" (which IIUC are go ver > 1.7) - happy to revert

Reply:
Happy to have it point to master; could you remove the comment though? It doesn't make sense outside of the context of the change, if that makes sense.

Reply:
Resolved in e9c5791